src/cli.cpp

```diff
@@ -114,17 +114,21 @@ int main(int argc, char *argv[])
             cout << "Generating plot for k=" << static_cast<int>(k) << " filename=" << filename
                  << " id=" << id << endl
                  << endl;
+            id = Strip0x(id);
```

Comment:
if there is a `0x` prefix, it should be stripped *before* checking that the length is 64 characters